wiki/People/The_Team/Nomination_Assessment_Team/Evaluations/en.md

```diff
@@ -0,0 +1,83 @@
+
+Warnings and being placed on probation due to subpar performance as a full Beatmap Nominator are effective for roughly one year. If a BN has reoccurring issues down the line after being warned or placed on probation recently, they may be removed from the Beatmap Nominators on future evaluations.
+
+## Special evaluations
+
+### Activity
+
+Beatmap Nominator activity is checked automatically by the NAT website on the 1st of every month. If it has found that there are nominators not meeting the activity requirements, they will be flagged and posted in their respective mode's NAT channel for NAT members to review by the "bnsite" Discord bot. Review will include whether the BN has posted an absence notice recently, and a double check to ensure the website check is accurate to the BN's activity status. 
+
+If it is found the BN is indeed not meeting activity, they will be either warned or removed depending on severity. Beatmap Nominators who were removed for activity in this way may apply again in 30 days, provided they have at least 8 mods to show they have come back to being active as a modder again.
+
+### Resignations
+
+Beatmap Nominators can resign from the group at any time via the NAT website. When this happens, three NAT members will be assigned to evaluate the nominator one last time, in order to determine if the resignation is on "good" terms or on "standard" terms. This evaluation is given a 24-hour deadline and announced to the NAT members in their respective Discord channel. Beatmap Nominators who resign on good terms may be able to fast-track back to being a full BN if they choose to apply again within 1 year of resignation. Those who resign on standard terms will have to start from probation if they pass their reapplication. 
+
+Generally, good- and standard-term evaluations are relatively simple. Beatmap Nominators who have not been recently warned or placed on probation, and were not about to be warned for issues, will usually resign on good terms. Standard terms is typically used for nominators who have been warned recently, are on probation, or were about to be warned for issues on their upcoming current BN evaluation.
+
+Once a consensus for the resignation terms has been reached, the NAT will send the BN a message via the [mappersguild bot](https://osu.ppy.sh/users/23648635) informing them of their status and when they can reapply to become a BN if they so choose.
+
+### Mock evaluations
+
+In addition to BN evaluations by NAT members, mock evaluations may also be given out to Beatmap Nominators for new BN application evaluations. In mock evaluations, nominators will submit individual evaluations alongside the NAT, but it will not determine the final result for the application unless the NAT evaluators cannot come to their own consensus. For the mock evaluation they participated in, the nominator will be able to read the individual evaluations in an anonymised format once the application is archived.
```

Comment:
```suggestion
In addition to BN evaluations by NAT members, mock evaluations may also be given out to Beatmap Nominators for new BN application evaluations. In mock evaluations, nominators will submit individual evaluations alongside the NAT, but it will not determine the final result for the application unless the NAT evaluators cannot come to their own consensus. Whilist NAT members are obliged to participate in the evaluation process, there are no consequences for nominators who do not fulfill the given assignment. For the mock evaluation they participated in, the nominator will be able to read the individual evaluations in an anonymised format once the application is archived.
```
Would add a word that BNs are fine if they don't fulfill the assignment.